internal/command/versions.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/cyberark/summon/pkg/summon"
+	"github.com/cyberark/summon/provider"
+)
+//Create the AllVersions string to include versions of all the providers
+const SUMMON_VERSION = summon.VERSION
+var DefaultPath = provider.DefaultPath
+var AllVersions = GetVersions(GetProviderFileNames())
+
+//Run `$provider --version`(from DefaultPath),
+// 	for each provider in the list of providers
+//Then appends the responded versions to string and returns it
+func GetVersions(providers []string) string{
+	var toReturn = SUMMON_VERSION
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(DefaultPath, provider), "--version").Output()
```

Comment:
don't do this but it's interesting to think about: if we are pulling all provider infos at the same time and we had a lot of providers, this could be done in parallel.